### PR TITLE
Translate README into Simplified Chinese

### PR DESCRIPTION
**Pull request by wayou**
- translate readme into simplified chinese
- improve the translation of rules

### CODE REVIEW

File: docs/README-esla.md [hunk 1]
```diff
@@ -17,9 +17,11 @@
 
 <p align="center">
   Translations:
-  <a href="../README.md">English</a>,
-  <a href="README-ptbr.md">Português</a>,
-  <a href="README-zhtw.md">繁體中文</a>
+  <a href="../README.md">English</a>、
+  <a href="README-ptbr.md">Português</a>、
+  <a href="README-zhtw.md">繁體中文</a>、
+  <a href="README-zhcn.md">简体中文</a>
```
Comment (feross) [inline docs/README-esla.md]: Can you leave the original commas?

File: docs/README-ptbr.md [hunk 2]
```diff
@@ -17,9 +17,11 @@
 
 <p align="center">
   Translations:
-  <a href="../README.md">English</a>,
-  <a href="README-esla.md">Spanish</a>,
-  <a href="README-zhtw.md">繁體中文</a>
+  <a href="../README.md">English</a>、
+  <a href="README-esla.md">Spanish</a>、
+  <a href="README-zhtw.md">繁體中文</a>、
+  <a href="README-zhcn.md">简体中文</a>
```
Comment (feross) [inline docs/README-ptbr.md]: Same here. Keep the `,` in place please.

**Comment by feross** [comment]
@wayou Thanks for the awesome contribution! I added you as a collaborator on this repository, so you can be part of the team (if you want). Please take a look at the contributing guidelines here: https://github.com/feross/standard/blob/master/CONTRIBUTING.md

There will probably be small changes to the readme and rules document over time and it would be great if you could help keep the Simplified Chinese translation up to date 👍 👍 👍

**Comment by feross** [comment]
Aaand, the changes are live here now! https://standardjs.com/readme-zhcn.html

**Comment by wayou** [comment]
nice too see :smile:

**Pull request merged**